### ISSUE TITLE
Replace example images with an aerial image

### ISSUE THREAD

**Issue by jywarren** [publiclab/Leaflet.DistortableImage#315]
The example images should really be aerial photos, looking downward, as usable on MapKnitter.org -- not oblique angles. I know we discussed this but let's please replace these so that they make more sense for actual map stitching -- here's an example image that's an appropriate size and has some nice visual features:

![image](https://user-images.githubusercontent.com/24359/60465968-6215e780-9c20-11e9-8c09-265e5ef9c1f6.png)

https://s3.amazonaws.com/grassrootsmapping/warpables/315613/img02_medium.jpeg

This should go in `/examples/index.html` as well as `/examples/select.html`

It's from https://mapknitter.org/maps/museum-park-01-rotterdam, and there are other images that could be used in `select.html` so that there are fewer of the same image, so it's a more realistic example:

https://s3.amazonaws.com/grassrootsmapping/warpables/315614/IMAG0179_r_medium.JPG

![image](https://user-images.githubusercontent.com/24359/60466141-d94b7b80-9c20-11e9-86c2-c25a54523f6f.png)

@rexagod this may be a good one for you to take on so that your auto-stitching demos are also running on real aerial photos! cc @sashadev-sky (though after our export work!)

**Comment by jywarren** [comment]
I'm so sorry! This is all good for the basic demo at `index.html`:

![image](https://user-images.githubusercontent.com/24359/60466445-a786e480-9c21-11e9-837b-0391cffccbe0.png)

I was looking at a cached version (so frustrating!) due to an unrelated project. But the above still goes for the multiple select demo!

**Comment by sashadev-sky** [comment]
@jywarren I was going to fix this didn't forget about it! I was holding off a bit because I wanted to discuss with you why we are not using the recommended "google mutant layers" which would potentially end up changing the image we are using for the demo

**Comment by sashadev-sky** [comment]
I agree the current demo makes it pretty confusing what is going on with the combo of the map and demo

**Comment by jywarren** [comment]
No prob, just making sure I also didn't forget! Thanks!

**Comment by sashadev-sky** [comment]
Ok cool! I sent you a slack message regarding the exporting in case you wanted to go through it again after the changes since I know it needs to be working asap

**Comment by sashadev-sky** [comment]
@jywarren ok so did you want, when this is fixed, for the map to also be updated to Google Mutant Layers?

**Comment by rexagod** [comment]
@jywarren Sure thing! I've replaced the older images with these in the demo. 👍

**Comment by jywarren** [comment]
Great! We should be all good here now!

**Comment by sashadev-sky** [comment]
@jywarren They are not updated for our demo? Did @rexagod mean his demo for the matcher? Also should I still open a PR to change the layers we use to google mutant layers?

**Comment by jywarren** [comment]
+1 google mutant layers, i guess... it's a bit more visually noisy but more
realistic!

I think I did update the demo --
https://publiclab.github.io/Leaflet.DistortableImage/examples/select do you
not see them? Cache maybe? Thanks!

**Comment by rexagod** [comment]
@sashadev-sky I was referring to the matcher test case for stitcher. Sorry for any confusion!
